modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -23,7 +23,9 @@ dex:
         memory: 
 ----
 
-. Configure Keycloak by editing the Argo CD CR, and updating the value for the `provider` parameter as `keycloak`. For example:
+. Configure Keycloak by editing the Argo CD CR, and updating the value for the `provider` parameter as `keycloak`.
```

Comment:
We can improvize this step to be more crisp and clear. We can remove unnecessary words, which are quite obvious if you go with the context and make this step more focused.
 
```suggestion
. Set the value of the `provider` parameter to `keycloak` in the Argo CD CR.
```